RestaurantReservation/RestaurantReservation.Db/Models/Views/EmployeeDetails.cs

```diff
@@ -0,0 +1,24 @@
+using RestaurantReservation.Db.Models.Enums;
+
+namespace RestaurantReservation.Db.Models.Views;
+
+public class EmployeeDetails
```

Comment:
It's good to follow this naming convention when dealing with views `Name + View` so it would be `EmployeeDetailsView`